openlibrary/plugins/ol_infobase.py

```diff
@@ -528,12 +528,7 @@ def normalize_edition_title(self, title):
         if not isinstance(title, six.text_type):
             return ""
 
-        # http://stackoverflow.com/questions/517923/what-is-the-best-way-to-remove-accents-in-a-python-unicode-string
-        def strip_accents(s):
-            return ''.join((c for c in unicodedata.normalize('NFD', s) if unicodedata.category(c) != 'Mn'))
-
-        norm = strip_accents(title).lower()
-        norm = norm.replace(' and ', ' ')
+        norm = title.lower().replace(' and ', ' ')
```

Comment:
Figuring out what this `norm` field is used for is needed, it looks like stripping accents is appropriate here, but I can't tell where this data goes.

I don't know how this would work for non-Latin characters, (it _may_ work for CJK / Hebrew / Arabic by stripping modifiers that might be catalogued differently?). Again, understanding what this normalisation is for would help.